packages/react-components/react-motion-preview/src/hooks/useMotion.test.ts

```diff
@@ -42,7 +42,10 @@ const renderHookWithRef = (
 
 const jumpAnimationFrame = () => {
   // requestAnimationFrame
-  act(() => jest.advanceTimersToNextTimer());
+  act(() => {
+    jest.advanceTimersToNextTimer();
```

Comment:
why is this called twice ? can we add some docs for this ?

Reply:
I added some comments to describe this need.